src/client/components/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,53 @@
+import './Autocomplete.scss'
+import React from 'react'
+
+import Turnstone from 'turnstone'
+
+type Props = {
+  // listbox: Object containing autocomplete options,
+  // mainly used for selecting correct key for displayName for object
+  listbox: Record<string, unknown>
+  disabled: boolean
+
+  value: string
+  name?: string
+  onChange: (value: any) => void
```

Comment:
can't you use generic instead of any ?

Reply:
Issue with `.tsx` files and generics, left as `value: string | any` as user input is always `string` in input text